### ISSUE TITLE
Give users a different path instead of subdomain

### ISSUE THREAD

**Issue by pheyvaer** [solid/node-solid-server#1349]
Is it currently possible to give different users a different path instead of subdomain.
If you have two users Ash and Misty, and the server is at `solid.example.org` then their PODs are at `ash.solid.example.org` and `misty.solid.example.org`. Is it possible to get something like the following instead? `solid.example.org/ash` and `solid.example.org./misty`.

**Comment by mikeadams1** [comment]
@pheyvaer IMHO, This is not a problem or an NSS issue, and I am against the idea in it's entirety.

**Comment by Ryuno-Ki** [comment]
Subdomains [can make a lot of sense in web programming](https://stackoverflow.com/q/1965609). Especially in a multi-tenant-like model like SoLiD's.

**Comment by jaxoncreed** [comment]
There is not anything in the solid spec that prohibits the use of alternative routing schemes. However, at the moment, NSS does not support that.

**Comment by melvincarvalho** [comment]
> Is it possible to get something like the following instead? solid.example.org/ash and solid.example.org/misty

Yes.  A WebID can live at any resource.  You can even have two in one file

You would need to put the WebID files at both those resources, and them permission then accordingly.  Not much difference from how you use your own file system

Do note that browsers have a bit of a quirky security model wrt sub domains aka origins.  For example cookies are not shared over different origins but they are over different paths in the same origin

I think the model you suggest works best if ash and misty are friends / family.  Are you asking if that process could be automated and put in as a configuration option?  If so, it may have some security considerations.

If not, is it OK to close this issue?

**Comment by pheyvaer** [comment]
But you can you use the Solid principles to edit the data of the WebID? Considering that the WebID is not hosted by a Solid POD.

**Comment by kjetilk** [comment]
> But can you use the Solid principles to edit the data of the WebID? Considering that the WebID is not hosted by a Solid POD.

No, you couldn't do that, but other uses should be OK.

BTW; as a tangent to this: My old university has a server `folk.uio.no` where all users can put their homepages. My academic homepage is for example http://folk.uio.no/kjekje/ It just occurred to me that it would be a reasonable upgrade path for all university users to have that as a Solid RS. 

The university would put a root acl on http://folk.uio.no/ that gave `acl:Read` to `foaf:Agent`, since that is what has been done there, and then users would opt in to having Solid by adding an acl to their own account giving themselves write access, and then they can go from there. The national acadmic SSO system, Feide, already supports OIDC, and should be the Solid IDP.

Back in the day, it used to be a lot of such academic servers around, perhaps it is an approach that could bring academia onto Solid quickly.

**Comment by pheyvaer** [comment]
Ok, but other files that are on the Solid POD are still under that subdomain, right?

The university use case is similar to what is happening at ours if I'm correct. So indeed might be a good approach to move to Solid!
